packages/components/src/root-container/utils.ts

```diff
@@ -0,0 +1,101 @@
+import { getKeys, PipeFunc, UnknownRecord } from '@devextreme/core';
+import {
+  Builder,
+  BuilderBase,
+  DomAttributes,
+  Props,
+} from './types';
+
+export function buildCss(
+  condition: boolean,
+  className: string,
+): PipeFunc<Props> {
+  return (props: Props) => (
+    condition
+      ? {
+        ...props,
+        cssClass: [
+          ...props.cssClass,
+          className,
+        ],
+      }
+      : props
+  );
+}
+
+export function buildAttributes(
```

Comment:
Question: Why it named build? It looks like conditional map?

Reply:
Renamed to map